src/app/shared/podcast-nav/podcast-nav.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Router } from '@angular/router';
+import { Store } from '@ngrx/store';
+import { Observable } from 'rxjs/Observable';
+import { Subscription } from 'rxjs/Subscription';
+import { PodcastModel, IntervalModel, FilterModel, ChartType } from '../../ngrx';
+import { selectFilter, selectPodcasts, selectSelectedPodcast } from '../../ngrx/reducers';
+
+@Component({
+  selector: 'metrics-podcast-nav',
+  template: `
+    <metrics-podcast-dropdown
+      [selectedPodcast]="selectedPodcast$ | async"
+      [podcasts]="podcasts$ | async"
+      (podcastChange)="onPodcastChange($event)"></metrics-podcast-dropdown>
+  `,
+  styleUrls: ['../menu/dropdown.css', './podcast-nav-dropdown.component.css']
+})
+export class PodcastNavComponent implements OnInit, OnDestroy {
+  filterSub: Subscription;
+  selectedInterval: IntervalModel;
+  selectedChartType: ChartType;
+  selectedPodcast$: Observable<PodcastModel>;
+  podcasts$: Observable<PodcastModel[]>;
+
+  constructor(private store: Store<any>,
+              private router: Router) {}
+
+  ngOnInit() {
+    this.filterSub = this.store.select(selectFilter).subscribe((filter: FilterModel) => {
```

Comment:
Part of me wishes there weren't any subscribes at all in here (just like ngrx promised!).  But another part of me doesn't know how to accomplish that in this case.

Reply:
Maybe make some `selectFilterInterval` and `selectFilterChartType` selectors, and then passing those down a level?  Hrmm... but I guess you'd have to move the routing down a level, which isn't ideal either.

Reply:
I do actually have a plan for getting rid of these. I think routing should be actions. Right now for example with the menu, I'm passing any routing up to the downloads component with EventEmitters, and I think those could be handled as actions by an effect rather than handled in a container component. https://github.com/PRX/metrics.prx.org/issues/138 Also I think filter being a RouteStateSerializer would be much cleaner.

Reply:
Yeah, that would make sense.  Good sense!